frontend_app_admin_workspace_user/src/css/index.styl

```diff
@@ -120,14 +120,21 @@
           color inherit
         & > i
           margin-right textSpacing
+        & .profileNavigation:hover
+          text-decoration underline
     &__active
       width 17%
     &__profile
-      width 62px // 62px it's the size of avatar's profile
+      width 49px // 49px it's the size of avatar's profile
```

Comment:
Can't we use a global css variable for the size of avatar's profile? Just like `bigIconSize` below?

Reply:
will fix

Reply:
fixed